hack/generate-crd-yaml.sh

```diff
@@ -15,8 +15,6 @@ cd $REPO
 # Controller-gen seems to use an unstable sort for the order of output of the CRDs
 # so, output them to separate files, then concatenate those files.
 # That should give a stable sort.
-# TODO(youngnick): We only do ./apis/projectcontour/... so that we can keep the IngressRoute definitions
-# around for a while. This removes them from the example CRDs for installers etc though.
 go run sigs.k8s.io/controller-tools/cmd/controller-gen \
   crd paths=./apis/projectcontour/... output:dir=$TEMPDIR
```

Comment:
If `./apis/contour/` is gone, then this can change back to `./apis/...`. I changed it this way to exclude the CRDs from generation.